lldb/include/lldb/Target/RegisterContextUnwind.h

```diff
@@ -67,6 +67,10 @@ class RegisterContextUnwind : public lldb_private::RegisterContext {
 
   bool ReadPC(lldb::addr_t &start_pc);
 
+#ifdef __AIX__
+  bool ReadLR(lldb::addr_t &lr);
+#endif
+
```

Comment:
It's likely we'd ask you to make this a method on all platforms that is only called on, or implemented for, AIX.